static/js/index.js

```diff
@@ -2425,16 +2425,9 @@ function loadScanAreaPolygons () {
 // MARK: - Filters
 
 function getPokemonSize (pokemonId, pokemonForm) {
-    const id = pokemonForm === 0 ? pokemonId : `${pokemonId}-${pokemonForm}`;
-    if (pokemonFilter[id] === undefined) {
-        // TODO: console.log('Pokemon size undefined:', id, 'form:', pokemonForm);
-        return 40;
-    }
+    const id = pokemonForm === 0 ? `${pokemonId}-${masterfile.pokemon[pokemonId].default_form_id}` : `${pokemonId}-${pokemonForm}`;
     const size = pokemonFilter[id].size;
-    if (size === 'huge') {
-        return 75;
-    }
-    return 40;
+    return (size === 'small' ? 20 : size === 'large' ? 60 : size === 'huge' ? 80 : 40);
```

Comment:
Minor complaint/suggestion: A switch/case or using an object as dictionary would be more readable.